README.md

```diff
@@ -20,6 +20,7 @@ class App extends Component {
           accessToken={`${YOUR_EMBED_TOKEN}`}
           filterPaneEnabled={false}
           navContentPaneEnabled={false}
+          pageName={`${YOUR_PAGE_ID}`}
```

Comment:
Could you change it to `pageId` instead of `pageName`?

Reply:
See comment here: https://github.com/howlowck/react-powerbi/pull/7#issuecomment-424288670

I tried, but it stopped it working.